src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="page__calendar calendar calendar--month-length--31">
```

Comment:
you never used `page__calendar` class
```suggestion
    <div class="calendar calendar--month-length--31">
```